charts/node-red/templates/deployment.yaml

```diff
@@ -25,6 +25,18 @@ spec:
       serviceAccountName: {{ include "node-red.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      initContainers:
```

Comment:
can you make this with a conditional? So if you dont want a persistence volume, it will not breake?

Reply:
fixed in f13a099